### PR TITLE
V1.2 add command

### LINKED ISSUE #1: As a user, I can add an exercise

### PR DESCRIPTION
**Pull request by Liu-2001**

### CODE REVIEW

**Review by aaronlwm — COMMENTED** [review]
Saw you commented out some tests, any reasons why?

**Comment by Liu-2001** [comment]
> Saw you commented out some tests, any reasons why?

Those tests are for AB3. The tests throw a lot of incompatible errors because I modified the original classes (AddCommand, AddCommandParser etc.) instead of creating new ones.

**Comment by Nauman-S** [comment]
update comments fpr exercise

**Comment by Royxuzeng** [comment]
Code looks good. I think it is good to merge.

**Comment by Nahoyhp** [comment]
Can't find any bugs. 
Good to go.

File: src/test/java/seedu/address/logic/parser/AddCommandParserTest.java [hunk 1]
```diff
@@ -1,85 +1,50 @@
...
     public void parse_allFieldsPresent_success() {
-        Person expectedPerson = new PersonBuilder(BOB).withTags(VALID_TAG_FRIEND).build();
+        Exercise expectedExercise = new ExerciseBuilder(PUSH_UP).build();
+
+        assertParseSuccess(parser, " n/Push Up d/Test 1 at/09-10-2020 c/12345", new AddCommand(expectedExercise));
 
         // whitespace only preamble
-        assertParseSuccess(parser, PREAMBLE_WHITESPACE + NAME_DESC_BOB + PHONE_DESC_BOB + EMAIL_DESC_BOB
-                + ADDRESS_DESC_BOB + TAG_DESC_FRIEND, new AddCommand(expectedPerson));
+        /*assertParseSuccess(parser, PREAMBLE_WHITESPACE + NAME_DESC_BOB + PHONE_DESC_BOB + EMAIL_DESC_BOB
```
Comment (aaronlwm) [inline src/test/java/seedu/address/logic/parser/AddCommandParserTest.java:22]: I think we can work on these now - see issue #37

**Pull request merged**